resources/user-agents/browsers/firefox/firefox-3-6.json

```diff
@@ -159,13 +164,6 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (#PLATFORM#; rv:1.9.2*) Gecko* Firefox *",
```

Comment:
I moved this to the generic file since it doesn't have #MAJORVER# or #MINORVER# tokens.